components/size_aware_image.jsx

```diff
@@ -178,18 +263,57 @@ export default class SizeAwareImage extends React.PureComponent {
 
             return (
                 <div
-                    onClick={this.handleImageClick}
-                    className={className}
-                    style={this.state.imageWidth > MIN_IMAGE_SIZE ? {
-                        width: this.state.imageWidth + 2, // 2px to account for the border
-                    } : {}}
+                    className={classNames('small-image-utility-buttons-wrapper')}
```

Comment:
More nitpicking, could you remove the use of `classNames` where the class name is a constant?